src/svgutils/transform.py

```diff
@@ -49,6 +49,45 @@ def rotate(self, angle, x=0, y=0):
         self.root.set("transform", "%s rotate(%f %f %f)" %
                       (self.root.get("transform") or '', angle, x, y))
 
+    def skew_x(self, angle):
```

Comment:
well, on a second thought it might be better to keep the transform interface close to SVG specs. So please keep `skew_x` and `skew_y`. Thanks!